fastlane.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/sliceutil"
+	"github.com/bitrise-steplib/steps-deploy-to-itunesconnect-deliver/appleauth"
+)
+
+// fastlaneAPIKey is used to serialize App Store Connect API Key into JSON for fastlane
+// see: https://docs.fastlane.tools/app-store-connect-api/#using-fastlane-api-key-json-file
+type fastlaneAPIKey struct {
+	KeyID      string `json:"key_id"`
+	IssuerID   string `json:"issuer_id"`
+	PrivateKey string `json:"key"`
+}
+
+// FastlaneParams are Fastlane command arguments and environment variables
+type FastlaneParams struct {
+	Envs, Args []string
+}
+
+// AppendFastlaneCredentials adds auth credentials to Fastlane envs and args
+func AppendFastlaneCredentials(inParams FastlaneParams, authConfig appleauth.Credentials) (FastlaneParams, error) {
```

Comment:
could not we just return the authentication related args and envs, without `inParams`?